src/contrib/mlir/compiler.cpp

```diff
@@ -225,7 +224,7 @@ void MLIRCompiler::build_ng_dialect_module()
     }
 
     // create builder
-    m_builder = llvm::make_unique<mlir::OpBuilder>(function.getBody());
+    m_builder = std::make_unique<mlir::OpBuilder>(function.getBody());
```

Comment:
Huh ?

Reply:
They have removed `llvm::make_unique` from LLVM since now LLVM supports C++14 and `std::make_unique` can be used instead.

Reply:
Currently ngraph is at c++11. Put it back to use llvm::make_unique or just make a unique_ptr